supernovacontroller/sequential/i3c.py

```diff
@@ -982,6 +992,7 @@ def ccc_unicast_disec(self, target_address):
 
         Args:
         target_address: The address of the target device on the I3C bus to which the DISEC command is directed.
+        events: A list containing events to be disabled.
```

Comment:
Please, mention that the type of element of the list belongs to the enum DISEC: DISEC.DISINT, DISEC.DISCR, DISEC.DISHJ. For instance [DISEC.DISINT, DISEC.DISHJ]